app/views/layouts/_flash_msg.html.haml

```diff
@@ -1,6 +1,7 @@
 - flash_div_id ||= 'flash_msg_div'
 
-%div{:id => flash_div_id, :style => ("display: none" unless @flash_array)}
+- flash_style ||= @flash_array ? "" : "display_none"
```

Comment:
typo: `display_none` is not a valid style, should be `display: none`

Reply:
=> https://github.com/ManageIQ/manageiq-ui-classic/pull/4540

please backport together :)